Modules/Role/Http/Services/RoleService.php

```diff
@@ -149,7 +144,8 @@ public function delete($id): array
     public function showAdmin(): array
     {
         $adminUser = User::join('roles', 'users.role', '=', 'roles.id')
-            ->select('users.first_name', 'users.last_name', 'users.email', 'roles.role', 'users.id')
+            ->where('roles.id','!=','1')
```

Comment:
Avoid join query. Make relationship of user model with roles and use the relation to get required data.

Reply:
Fixed